lib/rucio/common/objectstore.py

```diff
@@ -46,7 +46,7 @@
 
 REGION = make_region().configure('dogpile.cache.memcached',
                                  expiration_time=3600,
-                                 arguments={'url': "127.0.0.1:11211", 'distributed_lock': True})
+                                 arguments={'url': config.config_get('cache', 'url', True, '127.0.0.1:11211'), 'distributed_lock': True})
```

Comment:
the True should be False, otherwise we get an exception and not the default value ;-)

Reply:
Well that was silly. Should be fixed!

Reply:
oh no, github+travis are choking again and not starting the . the only reliable way to get it to run is to delete the pull requests, also delete the branches on your github fork, and then do tools/submit-pull-request again